<launch>
  <!-- Tester polaczen - ping -->
  <node pkg="robot_dhi_1" type="ConnectionTester.py" name="ConnectionTester" />
  <!-- Glowny program autodiagnostyki -->
  <node pkg="robot_dhi_1" type="AutoModeInitDiag.py" name="AutoModeInitDiag" output="screen"/>
  <!-- Program konwertujacy parametry jazdy dla sterownika PLC -->
  <node pkg="robot_dhi_1" type="DriveController.py" name="DriveController" output="screen"/>
  <!-- Program obslugujacy modul widel -->
  <!-- <node pkg="robot_dhi_1" type="ForkController.py" name="ForkController" /> -->
</launch>